backend/RayTracing/Classes/Models/AmbientLight.py

```diff
@@ -0,0 +1,15 @@
+class AmbientLight:
```

Comment:
Looks like a lot of unwanted files got added accidentally. Please only add the required files and push again. Thanks!